missing files in release tarball

Cockpit version: 236
OS: openSUSE Tumbleweed

The release tarball misses at least the following files preventing rebuild of the dist/ content:
```
  pkg/lib/cockpit-po-plugin.js
  pkg/lib/_global-variables.scss
```

Reproducer:
```
$ wget https://github.com/cockpit-project/cockpit/releases/download/236/cockpit-236.tar.xz
$ tar -xvf cockpit-236.tar.xz
$ cd cockpit-236/
$ rm -rf dist/
$ ./configure
$ npm install
$ make
...
Error: Cannot find module './pkg/lib/cockpit-po-plugin'
```

--- a/pkg/Makefile.am
+++ b/pkg/Makefile.am
@@ -18,6 +18,10 @@ pixmaps_DATA = pkg/sosreport/cockpit-sosreport.png
 EXTRA_DIST += \
 	pkg/users/mock \
 	pkg/lib/qunit-template.html \
+	pkg/lib/cockpit-po-plugin.js \
+	pkg/lib/included-modules-plugin.js \
+	pkg/lib/variables.scss \
+	pkg/lib/patternfly/patternfly-overrides-variables.scss \
 	$(pkg_TESTS) \
 	$(metainfo_DATA) \
 	$(pixmaps_DATA) \

